e2e/playwright/file-tree.spec.ts

```diff
@@ -1,4 +1,5 @@
-import { test, expect } from '@playwright/test'
+import { _test, _expect } from './playwright-deprecated'
+import { test, expect } from './fixtureSetup'
```

Comment:
so that we can have our old tests and the new fixtures exist in the same file.
`_test` are marked as deprecated in a doc string so if you're editor supports it they're strikes through
![image](https://github.com/user-attachments/assets/8d6e3b02-1f41-49a3-a63e-b6881ffd59d4)

But honestly it's not that hard to port ot the new fixtures so not sure how much we'll use this.